2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -160,6 +160,9 @@ collector {
       #brokers = "{{kafkaBrokers}}"
       ## Number of retries to perform before giving up on sending a record
       #retries = 0
+      #producerConf {
+      #  "compression.type"="snappy"
```

Comment:
When I remove all quotes like this: `compression.type = snappy` it results in: `"compression" -> "{"type":"snappy"}"`. Since the producer config key contains a dot it needs to be quoted.

Reply:
ah yes, too bad :/